lua/renamer/init.lua

```diff
@@ -375,16 +375,31 @@ function renamer._lsp_rename(word, pos)
             if resp.documentChanges then
                 changes = {}
                 for _, change in ipairs(resp.documentChanges) do
-                    changes[change.textDocument.uri] = {
-                        change.edits,
-                    }
+                    local edits = {}
+                    -- jdtls range indexing correction
+                    for _, edit in ipairs(change.edits) do
+                        table.insert(edits, {
+                            newText = edit.newText,
+                            range = {
+                                start = {
+                                    line = edit.range.start.line + 1,
+                                    character = edit.range.start.character + 1,
```

Comment:
Does this also need changing? I was under the impression columns were 1-indexed :thinking: